cmake/ci/oneflow-cuda-75.cmake

```
@@ -0,0 +1,19 @@
+set(BUILD_CUDA YES CACHE BOOL "")
+set(BUILD_TESTING YES CACHE BOOL "")
+set(BUILD_CPP_API YES CACHE BOOL "")
+set(WITH_MLIR YES CACHE BOOL "")
+set(BUILD_SHARED_LIBS YES CACHE BOOL "")
+set(CMAKE_LINK_DEPENDS_NO_SHARED YES CACHE BOOL "")
+set(THIRD_PARTY_MIRROR aliyun CACHE STRING "")
+set(PIP_INDEX_MIRROR "https://pypi.tuna.tsinghua.edu.cn/simple" CACHE STRING "")
+set(CMAKE_BUILD_TYPE Release CACHE STRING "")
+set(CMAKE_GENERATOR Ninja CACHE STRING "")
+set(CUDA_TOOLKIT_ROOT_DIR /usr/local/cuda CACHE STRING "")
+set(CUDNN_ROOT_DIR /usr/local/cudnn CACHE STRING "")
+set(CMAKE_CUDA_ARCHITECTURES "75" CACHE STRING "")
+set(CMAKE_C_COMPILER_LAUNCHER ccache CACHE STRING "")
+set(CMAKE_CXX_COMPILER_LAUNCHER ccache CACHE STRING "")
+set(CMAKE_CUDA_COMPILER_LAUNCHER ccache CACHE STRING "")
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION OFF CACHE BOOL "")
+set(CUDA_NVCC_THREADS_NUMBER 8 CACHE STRING "")
+set(WITH_TENSORRT YES CACHE BOOL "")
```

我们把这些文件内容复用一下吧，本文件可以改成

```suggestion
include(${CMAKE_CURRENT_LIST_DIR}/oneflow-cuda.cmake)
set(CMAKE_CUDA_ARCHITECTURES "75" CACHE STRING "")
```

---

好的